src/features/review/components/specimen-view/site-specimens.tsx

```diff
@@ -143,14 +145,40 @@ export function SiteSpecimenContent({
           </div>
 
           <div className="grid grid-cols-2 gap-4 md:grid-cols-4">
-            {specimens.map(specimen => (
-              <SpecimenImage
-                key={specimen.id}
-                specimen={specimen}
-                className="overflow-hidden rounded border transition-shadow hover:shadow-lg"
-                onClick={() => onImageClick(specimen)}
-              />
-            ))}
+            {specimens.map(specimen => {
+              const imageUrl = getSpecimenImageUrl(specimen);
```

Comment:
It seems SpecimenImage has been decoupled here. Is SpecimenImage being used anymore? If not, it should be deleted.